### DISCUSSION TITLE
Wrong Data from Inverter?

### DISCUSSION THREAD

**Discussion by WildEchoWanderer** [wlcrs/huawei_solar#383] [General]
I don't get it... I maybe have wrong Data from the Inverter. But i am not shure. Can you controll if i am just dumb or if something is Off:

![Screenshot_20230529_110604](https://github.com/wlcrs/huawei_solar/assets/116340711/9a02452d-48f3-4f04-84cc-a1da00cdb2fc)
This is the Inverter Dashboard in Fusion Solar. Exactly when my battery is full, the Input graph of the PV gets smaller around 1 kW. Before i had the battery, it was smooth and had usually not this Big values change. Only when it was cloudy/sunny. 
The Same is visible in Homeassistant:
![Screenshot_20230529_111638](https://github.com/wlcrs/huawei_solar/assets/116340711/0907aace-6ba7-429d-b02c-63e947f4baa7)

This was the weather
![Screenshot_20230529_112328](https://github.com/wlcrs/huawei_solar/assets/116340711/d22e4bdb-f705-4b63-8e07-facce3c3f298)

Anyone can Spot the error? Or is this the weather? Do the Battery load somehow add in the Inverter?

**Comment by WildEchoWanderer** [comment]
![Screenshot_20230529_134322_com huawei smartpvms](https://github.com/wlcrs/huawei_solar/assets/116340711/78b7b9e6-a749-4dc3-93a0-dd89253863f5)
Update from today. I started the tumbler dryer & washing Maschine and it looks Like this

**Comment by jires** [comment]
This is discussion about integration of Huawei Inverters into HA, and not really helpdesk for Huawei PV Systems, so your post is quite of topic...
But anyway, it looks like you have active power limiting set up in the fusion solar.. Check what entity Device state shows during the day.

**Comment by WildEchoWanderer** [comment]
Just for anybody wondering how i solved it:
I updated all the firmwares to the newest possible. they were hard to find
